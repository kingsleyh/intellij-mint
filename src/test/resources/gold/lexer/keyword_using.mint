record B {
  name : String using "first_name",
  age : Number using "my_age"
}