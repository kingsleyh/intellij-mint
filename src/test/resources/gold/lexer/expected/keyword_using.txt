record ('record')
WHITE_SPACE (' ')
CAPITALISED_IDENT ('B')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n  ')
LOWER_IDENT ('name')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
CAPITALISED_IDENT ('String')
WHITE_SPACE (' ')
using ('using')
WHITE_SPACE (' ')
STRING_LITERAL ('"first_name"')
, (',')
WHITE_SPACE ('\n  ')
LOWER_IDENT ('age')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
CAPITALISED_IDENT ('Number')
WHITE_SPACE (' ')
using ('using')
WHITE_SPACE (' ')
STRING_LITERAL ('"my_age"')
WHITE_SPACE ('\n')
} ('}')